pkg/bitcoin/estimator_test.go

```diff
@@ -0,0 +1,107 @@
+package bitcoin
+
+import (
+	"github.com/keep-network/keep-core/pkg/internal/testutils"
+	"reflect"
+	"testing"
+)
+
+func TestTransactionSizeEstimator_VirtualSize(t *testing.T) {
+	var tests = map[string]struct {
+		estimator           *TransactionSizeEstimator
+		expectedVirtualSize int
+		expectedError       error
```

Comment:
The `TransactionSizeEstimator.VirtualSize` method is forced to return an error as the estimator uses some functions from `bitcoin/script.go` that may return an error in some cases. In the current implementation, the estimator uses those functions in a way that will never produce an error, but still, we need to reflect that fact in the API and in the unit tests as this may change in the future. The `expectedError` in those tests is always `nil` so I think it should be here to emphasize this fact.